app/controllers/admins/orders_controller.rb

```diff
@@ -3,7 +3,19 @@
 module Admins
   class OrdersController < Admins::BaseController
     def index
-      render :index, locals: { orders: Order.includes(:employee) }
+      render :index, locals: { orders: Order.includes(:employee).order(status: :asc) }
+    end
+
+    def change_status
+      order = Order.find(params[:id])
+      order.update(status: 'delivered')
```

Comment:
you could use enum api to update te status :) https://api.rubyonrails.org/v5.2.4.4/classes/ActiveRecord/Enum.html